src/midi/midi_mt32.cpp

```diff
@@ -527,11 +527,17 @@ static mt32emu_report_handler_i get_report_handler_interface()
 		}
 
 		static void printDebug([[maybe_unused]] void* instance_data,
-		                       const char* fmt, va_list list)
+		                       [[maybe_unused]] const char* fmt,
+		                       [[maybe_unused]] va_list list)
 		{
-			char msg[1024];
-			safe_sprintf(msg, fmt, list);
-			LOG_DEBUG("MT32: %s", msg);
+#if !defined(NDEBUG)
+			// Skip processing MT-32 debug output in release builds
+			// because it can be bursty
+			char msg[1024] = "MT32: ";
+			assert(fmt);
+			safe_strcat(msg, fmt);
+			LOG_DEBUG(msg, list);
+#endif
```

Comment:
So do we understand how the debug logging resulted in a crash? Can't be a buffer overflow because of `safe_strcat`, can it?

Reply:
Yes; the VA list contained some symbols and junk values that weren't handled properly. I could have added more error checking, but in the end we had two formatting passes here (our own then logurus); so I opted to let Loguru do the VA list formatting (as it has all the return values checks from vsnprintf).

Reply:
Thanks for the explanation @kcgen 👍🏻